python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -162,7 +179,7 @@ def setUp(self, gangadir=None, wipe_repo=None, extra_opts=[]):
         # This is called before each unittest
         if gangadir is None:
             import os
-            gangadir = os.path.join('$HOME/gangadir_testing', self.__class__.__name__)
+            gangadir = os.path.join('$HOME/gangadir_testing')  # TODO make separate dirs per test suite, `self.__class__.__name__`
```

Comment:
In the near future I'd like to switch to using pytest as our test runner which has [the option](https://pypi.python.org/pypi/pytest-xdist) of running each test in an isolated subprocess. I think this wil help fix a lot of these test problems we're seeing. I've got a branch working with a PR incoming.